authenticatorbridge/src/main/java/com/bitwarden/authenticatorbridge/manager/model/AuthenticatorBridgeConnectionType.kt

```diff
@@ -0,0 +1,28 @@
+package com.bitwarden.authenticatorbridge.manager.model
+
+import com.bitwarden.authenticatorbridge.manager.AuthenticatorBridgeManager
+
+/**
+ * Models different connection types for [AuthenticatorBridgeManager].
+ */
+enum class AuthenticatorBridgeConnectionType {
+
+    /**
+     * Connect to release build variant.
+     */
+    RELEASE,
+
+    /**
+     * Connect to dev build variant.
+     */
+    DEV,
+}
+
+/**
+ * Convert a [AuthenticatorBridgeConnectionType] to raw package name for connection.
+ */
+internal fun AuthenticatorBridgeConnectionType.toPackageName() =
+    when (this) {
+        AuthenticatorBridgeConnectionType.RELEASE -> "com.x8bit.bitwarden"
+        AuthenticatorBridgeConnectionType.DEV -> "com.x8bit.bitwarden.dev"
+    }
```

Comment:
👍

Reply:
I'm curious what @david-livefront thinks but we don't usually mix classes and extension functions for that class in a single file, instead preferring to have a `util` package to place these in with a `....Extensions` file.

Reply:
Sure thing: a013a30d5c6c7b38c87d7d35ed3861264415f829

Reply:
Yea, an `AuthenticatorBridgeConnectionTypeExtensions` file sounds good, Looks like it's already done!